products/jse-dept/features/jse_dept_related_content.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Ensure Site Users can view, select, and reset the filter by related content section on the news and events page
+  In order to ensure that any site user can view, select, and reset the filter by related content section on the news and events page
+  As a Site User
+  I want to be able to view, select, and reset the filter by related content section on the news and events page
+
+  Background:
+     Given I am on "News & Events"
```

Comment:
@cjwest I'm actually getting a little confused here. Some sites have a single News & Events page while others have a separate News page and Events page. Since these tests are department sites tests, what format does department sites follow? In the "https://sites.stanford.edu/jse-prod/" that you created, the news and events page are separated but the News page doesn't have a filter by related content section. I'm getting confused on what the requirements for the particular site are